client/components/search-list.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {List, ListItem} from 'material-ui/List'
+import Divider from 'material-ui/Divider'
+import Subheader from 'material-ui/Subheader'
+import Avatar from 'material-ui/Avatar'
+import {lightBlack} from 'material-ui/styles/colors'
+
+export class SearchList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { list: [] }
+  }
+
+  render() {
+    console.log(this.props.list)
+    return (
+      <div>
+        <List>
+          <Subheader>Search Results</Subheader>
+          {
+            this.props.list.map((channel, i) => {
+              return <div key={i}>
+                <ListItem
+                  leftAvatar={<Avatar src={channel.logo} />}
+                  primaryText={
+                    <p>{channel.display_name}&nbsp;&nbsp;<span style={{color: lightBlack}}>Followers: {channel.followers}</span></p>
```

Comment:
instead of `nbsp` can you use padding? Things like this can introduce some gnarly styling bugs.